modules/microshift-greenboot-healthchecks-list.adoc

```diff
@@ -0,0 +1,18 @@
+// Module included in the following assemblies:
+//
+// * microshift_running applications/microshift-greenboot.adoc
+
+:_content-type: CONCEPT
+[id="list-health-checks-greenboot_{context}"]
+== List of health checks
+//Q: is this content correct, or do we have another list for microshift?
```

Comment:
I do not think we should replicate this in our docs. Why not to refer to greenboot documentation?
In this section, we should note that this package needs to be installed explicitly for enable the checks.

Reply:
Users do not like being referred outside the primary docs they are in, so I can link to it, but it isn't a best practice. Do we have a procedure to give for installing this piece?

Reply:
Yes, we already list MicroShift-specific health checks in the intridcution, using table format

Reply:
The installation is implicit, once microshift-greenboot package is bundled into the ISO